tests/apps_test.go

```diff
@@ -204,7 +204,8 @@ var _ = Describe("deis apps", func() {
 				Eventually(sess).Should(Exit(0))
 			})
 
-			Specify("that user can run a command with quotes in that app's environment", func() {
+			// FIXME: why is Workflow failing this test?
```

Comment:
Assuming this PR passes--that the `FIXME:` is needed--I'll create an issue to track it before I merge this.